records/SMP_adding_translation.md

```diff
@@ -0,0 +1,6 @@
+# Process of adding a new string using the Translation component in sm_platform
+When changes are made with a string that uses the Translate component or translate() function in a PR, file a ticket or a follow up PR to ensure that the correspoding Chinese and Russian language files are updated as well.
```

Comment:
Whose responsibility is it to file the ticket? The person making the original change?